pkg/agent/config/node_config.go

```diff
@@ -18,6 +18,8 @@ import (
 	"fmt"
 	"net"
 
+	"github.com/rakelkar/gonetsh/netroute"
```

Comment:
I think this library is only available on Windows. But it looks you have added it in the generic files. Will it block the compilations on Linux?

Reply:
Thanks Wenying, remove the import in update.

Repleace "[]netroute.route" with "[]interface{}".